crates/uv-interpreter/src/python_query.rs

```diff
@@ -344,8 +344,8 @@ mod windows {
             .in_scope(|| Command::new("py").arg("--list-paths").output())
             .map_err(Error::PyList)?;
 
-        // There shouldn't be any output on stderr.
-        if !output.status.success() || !output.stderr.is_empty() {
+        // `py` sometimes prints "Installed Pythons found by py Launcher for Windows" to stderr which we ignore.
```

Comment:
That's funny that it only does this sometimes. I tried to reproduce this on my machine with the official `py` launcher and never managed to get it write to stderr. 

I don't mind the change but we may want to log the `stderr` output with `tracing::debug!()` if it's non empty. It might be helpful when debugging issues related to `py`.

Reply:
![image](https://github.com/astral-sh/uv/assets/403995/a4f538a8-0bbc-4aae-909b-46094264f2d0)

It's consistent on my machine; I meant "sometimes" as in depending on which windows box you're on.

My `py` does this in a command window as well as bash (mingw).